apps/ern-reconnet/src/views/home-page.vue

```diff
@@ -39,15 +39,9 @@
         <LinkCard id="quicklink-documents" :imageSrc="DocumentsImage">
           <router-link :to="{ name: 'documents' }">Documents</router-link>
         </LinkCard>
-        <LinkCard id="quicklink-dashboard">
+        <LinkCard id="quicklink-dashboard" :imageSrc="DashboardImage">
```

Comment:
use direct asset loader ?<img src="@/assets/ern-reconnet-dashboard-header.jpg"/>

Reply:
I will fix this in the next release.